src/components/PastMeetups.js

```diff
@@ -1,75 +1,92 @@
-import Image from 'gatsby-image'
-import { Box, Heading, Paragraph, Text, Button } from 'grommet'
-import { Rewind } from 'grommet-icons'
-import React from 'react'
-import { theme } from './theme'
-import { FullWidth } from './TwoCols'
-import { graphql, useStaticQuery, Link, navigate } from 'gatsby'
+import React, { useContext } from 'react'
+import { Box, Heading, Text, Image, ResponsiveContext } from 'grommet'
+import { Calendar } from 'grommet-icons'
+import { graphql, useStaticQuery, navigate } from 'gatsby'
+import styled from 'styled-components'
 import Time from './Time'
 
-const colors = theme.global.colors
+const StyledH3 = styled(Heading)`
```

Comment:
please use the global theme props to style individual headings: https://v2.grommet.io/heading#heading.level